From 2f6a65718a0992d0624836ce4b1416a7420782d2 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Mon, 25 Jul 2016 22:52:41 +0200
Subject: [PATCH] lib: remove *.eo.h files from includesub_HEADERS

Otherwise *.eo.h files will be installed twice, so it can break a
parallel install on fast machines, whereby two make jobs may run
concurrently, trying to install the same files.

/usr/bin/install -c -m 644 elm_sys_notify.eo.h
/usr/bin/install -c -m 644 elm_sys_notify.eo.h
usr/bin/install: cannot create regular file 'elm_sys_notify.eo.h': File exists

Fixes:
http://autobuild.buildroot.net/results/7e9/7e9caffe442dcf40ee41da56fd3f55ffedf3a291

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 src/lib/Makefile.am | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/lib/Makefile.am b/src/lib/Makefile.am
index 40fb597..9df9424 100644
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -351,8 +351,6 @@ elm_systray_eo.h \
 elm_systray_common.h \
 elm_systray_watcher.h \
 elm_sys_notify.h \
-elm_sys_notify.eo.h \
-elm_sys_notify_dbus.eo.h \
 elm_table.h \
 elm_table_eo.h \
 elm_table_legacy.h \
@@ -381,7 +379,6 @@ elm_win_common.h \
 elm_win_eo.h \
 elm_win_legacy.h \
 elm_win_standard.h \
-elm_win_standard.eo.h \
 elm_helper.h
 includesubdir = $(includedir)/elementary-@VMAJ@/
 
-- 
2.5.5

